client/src/pages/Edit.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from "react";
+import Button from "@mui/material/Button";
+import styled from "styled-components";
+import { useNavigate } from "react-router-dom";
+import axios from "axios";
+
+function Edit() {
+  const navigate = useNavigate();
+  const [editFood, setEditFood] = useState({
+    foodName: "",
+    foodAuthor: "",
+    img: "",
+  });
+
+  useEffect(() => {
+    const editFoodId = async () => {
+      const reqdata = await fetch("http://localhost:5000/food");
+      const response = reqdata.json();
+      console.log(response);
+    };
+    editFoodId();
+  }, []);
+
+  const onInputChange = (e) => {
+    setEditFood({ ...editFood, [e.target.name]: e.target.value });
+  };
+
+  const FormHandle = (e) => {
+    e.preventDefault();
+    const res = axios.post("http://localhost:5000/food", editFood);
```

Comment:
How do you feel about using your new `fetch` code Instead of `axios` here?

```suggestion
    const response = await fetch("http://localhost:5000/food", {
      method: 'POST',
      body: JSON.stringify(editFood),
      Content-Type: 'application/json',
    });
      const result = {
        data: null,
        error: null,
      };
      if (response.ok) {
        result.data = await response.json();
      } else {
        result.error = await response.text();
      }
      return result;
```

As always, review code is untested, written from memory, and may need some fine-tuning before it works correctly.